### PR TITLE
Replace `Lato` font to system font

### PR DESCRIPTION
**Pull request by a-chabin**
We looked with the designers that the font has bugs on Windows and  Mac OS Safari (non-retina screen). The designers and I decided that we don't need custom fonts (except monospaced).

- Deleted font files
- Corrected `font-weight` (800 -> 700)
- Deleted postcss-mixins
- Reduced the size-limit

**Bugs**

### CODE REVIEW

**Comment by a-chabin** [comment]
waiting for the approval from the designers

**Review by ai — APPROVED** [review]

**Pull request merged**
